testah-junit/src/main/java/org/testah/framework/testPlan/AbstractTestPlan.java

```diff
@@ -33,22 +31,26 @@
 import org.testah.framework.cli.TestFilter;
 import org.testah.framework.dto.StepAction;
 import org.testah.framework.dto.TestDtoHelper;
+import org.testah.runner.TestahJUnitRunner;
 import org.testah.runner.testPlan.TestPlanActor;
 
+import java.lang.reflect.Method;
+import java.util.HashMap;
+
 /**
  * The Class AbstractTestPlan.
  */
 @ContextHierarchy({ @ContextConfiguration(classes = TestConfiguration.class) })
 public abstract class AbstractTestPlan extends AbstractJUnit4SpringContextTests {
 
     /** The test plan. */
-    private static ThreadLocal<TestPlanDto> testPlan;
+    private static ThreadLocal<TestPlanDto> testPlan = new ThreadLocal<>();;
```

Comment:
duplicate ;; at end of line, TestCaseDto and TestStepDto, in the next few lines as well.